.gitignore

```diff
@@ -1,2 +1,3 @@
 xcuserdata
 project.xcworkspace
+.DS_Store
```

Comment:
> Don't commit the .DS_Store files. You should add it to your [**global gitignore**](https://www.google.com/search?q=global+gitignore&oq=global+gitignore&aqs=chrome.0.35i39j0l5.2375j0j1&sourceid=chrome&ie=UTF-8) so they're not accidentally committed.

Reply:
I've removed the .DS_Store file and added it to .gitignore